src/main/java/com/cloudogu/smeagol/wiki/domain/Email.java

```diff
@@ -0,0 +1,62 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import com.google.common.base.Preconditions;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+/**
+ * Email address value object.
+ */
+public final class Email {
+
+    // TODO should we use such a strong validation?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1135)